src/Tizen.NUI/src/internal/WebView/WebCookieManager.cs

```diff
@@ -108,5 +117,17 @@ public void ClearCookies()
             Interop.WebCookieManager.ClearCookies(SwigCPtr);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
+
+        /// <summary>
+        /// Registers callback for changing watch.
+        /// <param name="callback">callback for changing watch</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void RegisterChangesWatchedCallback(ChangesWatchedCallback callback)
```

Comment:
What does it mean "Changes watched"?
"Changed detected" or "Watch changed"?

Reply:
If it is similar with [this](https://developer.chrome.com/docs/extensions/reference/cookies/#event-onChanged), I think it should be an event and the `Changed` is better name.